libs/types/src/index.ts

```diff
@@ -6,6 +6,8 @@ export * from './cast_vote_record';
 export * from './converter_client';
 export * from './dom';
 export * from './election';
+export * as ElectionADT from './election_adt';
```

Comment:
I wondered why this was being allowed. Turns out it's a bug: https://github.com/votingworks/vxsuite/pull/2995.